docs/changes/v5.6.0/Parameter-Changes.adoc

```diff
@@ -0,0 +1,7 @@
+= Paramater Changes =
+
+== enableOCSP become enableRevocationCheck ==
+
+Since verification with CRL-DP is introduced and it is enabled using
+this parameter in `server.xml` connector, the name is modified to be not related only to OCSP.
```

Comment:
Since we're going to keep `enableOCSP`, I'd suggest to modify this to be something like:

Rename enableOCSP to enableRevocationCheck

The enableOCSP param has been deprecated. Use enableRevocationCheck instead.